### PR TITLE
[#12048] Migrate RemindFeedbackSessionSubmissionAction

### PR DESCRIPTION
**Pull request by kevin9foong**
Migrated and wrote unit tests

### CODE REVIEW

File: src/main/java/teammates/ui/webapi/RemindFeedbackSessionSubmissionAction.java [hunk 1]
```diff
@@ -20,34 +22,63 @@ void checkSpecificAccessControl() throws UnauthorizedAccessException {
         String courseId = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ID);
         String feedbackSessionName = getNonNullRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
 
-        FeedbackSessionAttributes feedbackSession = getNonNullFeedbackSession(feedbackSessionName, courseId);
+        if (isCourseMigrated(courseId) && isAccountMigrated(userInfo.getId())) {
```
Comment (daongochieu2810) [inline src/main/java/teammates/ui/webapi/RemindFeedbackSessionSubmissionAction.java:25]: no need to check for both course and account regarding migration, you can assume that account is migrated first. The flow should be like this: check if account is migrated -> if yes, check if course is migrated; if no, course is not migrated, carry out the old logic

**Review by daongochieu2810 — CHANGES_REQUESTED** [review]

File: src/main/java/teammates/ui/webapi/RemindFeedbackSessionSubmissionAction.java (hunk 1 above)
Comment (kevin9foong) [reply src/main/java/teammates/ui/webapi/RemindFeedbackSessionSubmissionAction.java:25]: Does this mean to swap the ordering of the check with Account migrated, then Course migrated?

Comment (kevin9foong) [reply src/main/java/teammates/ui/webapi/RemindFeedbackSessionSubmissionAction.java]: Fixed and resolved

**Review by daongochieu2810 — APPROVED** [review]

**Review by samuelfangjw — APPROVED** [review]

**Comment by kevin9foong** [comment]
> @kevin9foong To merge this PR, would it be possible to update this branch with the latest `v9-migration`? Thanks!

@jasonqiu212 updated

**Comment by kevin9foong** [comment]
hi @samuelfangjw, could you please help me merge this into the v9-migrations?

**Pull request merged**